src/executors/default.yml

```diff
@@ -8,4 +8,4 @@ parameters:
   tag:
     description: Specify the Terraform Docker image tag for the executor
     type: string
-    default: "1.0.0" # update commands/install when updating this
+    default: latest
```

Comment:
Could we leave this as 1.0.0 for the time being, that we can include this change as a feature change in the next update coming soon? We can update this to latest in the next major version change we will issue shortly after.

Reply:
done!